components/brave_vpn/browser/DEPS

```diff
@@ -1,3 +1,5 @@
 include_rules = [
+  "+chrome/elevation_service",
```

Comment:
these are layering violations, you cannot have `chrome/*` in components

Reply:
Fixed in https://github.com/brave/brave-core/commit/6f77a96a2c5e41feedf1946827c5b97f3429233d